opam/solo5-bindings-spt.opam

```diff
@@ -10,13 +10,14 @@ bug-reports: "https://github.com/solo5/solo5/issues"
 license: "ISC"
 dev-repo: "git+https://github.com/solo5/solo5.git"
 build: [
-  ["./configure.sh"]
-  [make "V=1" "CONFIG_HVT=" "CONFIG_VIRTIO=" "CONFIG_MUEN=" "CONFIG_GENODE=" "CONFIG_XEN="]
+  ["./configure.sh" "--prefix=%{prefix}%" "--enable-targets=spt"]
+  [make "V=1"]
 ]
-install: [make "V=1" "CONFIG_HVT=" "CONFIG_VIRTIO=" "CONFIG_MUEN=" "CONFIG_GENODE=" "CONFIG_XEN=" "install-opam-spt" "PREFIX=%{prefix}%"]
+install: [make "V=1" "install-bindings" "install-tenders"]
 depends: [
-  "conf-pkg-config"
-  "conf-libseccomp" {os = "linux"}
```

Comment:
I think `conf-libseccomp` is still necessary for the spt bindings as it's needed to install the `libseccomp` external dependency.

Reply:
Correct. Fixed.